Sources/Citadel/TTY/Client/TTY.swift

```diff
@@ -252,21 +267,29 @@ extension SSHClient {
             return createChannel.futureResult
         }.get()
         
-        // We need to exec a thing.
-        let execRequest = SSHChannelRequestEvent.ExecRequest(
-            command: command,
-            wantReply: true
-        )
         
-        try await channel.triggerUserOutboundEvent(execRequest)
+        let channelRequest: Any
```

Comment:
Can we directly emit the channelRequest here? I don't like storing anything in an existential if possible, and I don't think we have to